themes/docs-new/assets/js/swiftype-config.js

```diff
@@ -0,0 +1,217 @@
+  https://swiftype.com/documentation/site-search/guides/faceted-search
+*/
+
+$(document).ready(function() {
+
+  // Swiftype UI elements //
+
+  var showSearchModal = function() {
+    $("#swiftype-modal").show(500);
+    $("#swiftype-search-form-modal").focus();
+  }
+
+  var hideSearchModal = function() {
+    $("#swiftype-modal").hide(250);
+    $("#swiftype-custom-facets input:checkbox[class='product-filter']").prop( "checked", false );
+    window.location.hash = "";
+    $(".swiftype-search-input").val('');
+    $("div.swiftype-widget > div.autocomplete").html("<ul></ul>");
+  }
+
+  $("#swiftype-close-button").click(function(){
+    hideSearchModal();
+  });
+
+  $('body').click(function (event) {
+    if(!$(event.target).closest('#swiftype-modal-content').length && !$(event.target).is('#swiftype-modal-content')) {
+      hideSearchModal();
+    }
+  });
+
+  $("#swiftype-custom-facets-toggle-button").click(function(){
+    if ($(window).width() < 768) {
+      $("#swiftype-custom-facets").toggle(500);
+      $("#swiftype-custom-facets-toggle-caret").find('svg').toggle();
+    }
+  });
+
+  $('#swiftype-facet-infra').on('change',function(){
+    $('#swiftype-facet-client, #swiftype-facet-server').prop('checked',$(this).prop('checked'));
+  });
+
+  $(window).resize(function() {
+    if ($(this).width() < 768) {
+      $('#swiftype-custom-facets').hide();
+    } else {
+      $('#swiftype-custom-facets').show();
+      }
+  });
+
+  const searchHashRegex = /^#stq./;
+  const searchStringMatchRegex = /^#stq=(.*)/;
+  const searchPageNumRegex = /&stp=\d{1,3}$/;
+
+
+  var processLocationHash = function() {
+    var hash = searchStringMatchRegex.exec(window.location.hash)[1];
+    pageNumTrue = searchPageNumRegex.exec(hash);
+    if (pageNumTrue) {
+      var matchString = decodeURIComponent(hash.substring(0,pageNumTrue.index));
+    } else {
+      var matchString = decodeURIComponent(hash);
+    };
+    $('input#swiftype-search-form-modal').val(matchString);
+  };
+
+  // Watch for hashchange
+  // #stq=[whatever]&stp=[number]
+  // #stq=release notes&stp=1
+  // it matches #stq
+  $(window).on('hashchange', function() {
+    if (searchHashRegex.test(window.location.hash)) {
+      processLocationHash();
+      showSearchModal();
+    }
+  });
+
+  if (searchHashRegex.test(window.location.hash)) {
+    processLocationHash();
+    showSearchModal();
+  };
+
+  // Autocomplete UI elements
+  $("input.swiftype-search-input").on('keypress', function (event) {
+   if (event.keycode == 13 || event.which == 13) {
+      event.preventDefault();
+      window.location.hash = 'stq=' + encodeURIComponent($(this).val()) + '&stp=1';
+    };
+  });
+
+  /////////////////////////
+  // Handle Search Results
+  /////////////////////////
+  var searchConfig = {
+    "facets": {}
+  };
+
+  /////////
+  // Handle Regular Search Results
+  /////////
+  var resultTemplate = Hogan.compile([
+    "<a class='swiftype-result' href='{{url}}'>",
```

Comment:
Part of the comment below regarding using an unordered list `ul` element.  I would wrap the `a` tag here in an `<li>`